README.md

```diff
@@ -27,14 +27,23 @@ The Lounge is the official and community-managed fork of [Shout](https://github.
 ## Installation and usage
 
 The Lounge requires [Node.js](https://nodejs.org/) v4 or more recent.
+[Yarn package manager](https://yarnpkg.com/) is also required to install The Lounge.
 
-### Running stable releases from npm (recommended)
+*While using `npm` might work, we no longer officially support this method.*
```

Comment:
I'm not sure how I feel about dropping support entirely for npm. This feels drastic.

Reply:
What? I was following along thinking this was just for the package system (`lounge install` etc.). I'd need two Node package managers installed if this goes through as-is?

Reply:
It's not so much about dropping entirely npm support, mainly switching the docs to `yarn`. As you can see in this PR, `package.json` does not change, so everything _should_ work the same way with `npm`.
However, I don't see myself testing everything we do in 2 different folders, with 2 different setups to make sure all new changes (dependencies, etc.) are 100% compatible. So yeah, official documentation to suggest `yarn`, and if people want to keep using `npm`, that's fine, they still can. If a bug arises that's only with `npm`, we can try to fix it, but that's about it, we won't set up 2 CIs, etc.

Also @dgw, I was fairly skeptical about `yarn` not long ago for that exact same reason, needing 2 package managers for Node, but as I started using it, I could see how `yarn` was much much superior to `npm`. But again, you don't have to use `yarn` with TL v3, it's just how it'll be documented.

Reply:
I didn't want to switch to Yarn either, but different npm issues and their management just pushed me to switch.

Reply:
Right, but we are conflating things here.

We are switch to yarn for our CI and for the internal package manager.

However, how people install our app shouldn't change. We shouldn't be forcing them to change to yarn if they don't want to. I know we aren't purposefully breaking npm, but I very much disagree with saying `While using `npm` might work, we no longer officially support this method.`.

We definitely *should* be supporting npm, and ensuring it still works.

Reply:
We shouldn't be able to break npm because we're only using it for 2 things: installing lounge (out of our control in the first place) and starting lounge. Both of these actions are compatible with yarn. 

Supporting and ensuring npm works would mean testing it on CIs, no one is going to be testing it manually.

Reply:
> We shouldn't be able to break npm because we're only using it for 2 things: installing lounge (out of our control in the first place) and starting lounge. Both of these actions are compatible with yarn.

Yeah, exactly, so this line is also unnecessary.

> Supporting and ensuring npm works would mean testing it on CIs, no one is going to be testing it manually.

No, we don't. We support all the features in our app but that doesn't mean they are all tested on CIs. What support means is that if someone finds a bug with something, we'll probably fix it. And we won't use any yarn specific things for what you would need for npm.

Also, of course someone is going to be testing it manually. @dgw has already said that he doesn't want to use yarn. I still use npm for the majority of things, I won't be moving to yarn for the lounge anytime soon.